rubot.rb

```diff
@@ -110,6 +113,41 @@ def handle(event_type, payload)
   end
 end
 
+# This isn't as perfect but gets whatever done.
+def check_config(event_type, payload)
+  event_type = event_type.delete('_')
+  payload = WSPayload.new(payload)
+  if $config["showclosed"]
+    case payload.action
+    when "closed"
+      return false
+    when "unlabeled"
+      return false 
+    when "unassigned"
+      return false
+    end
+  end
+  if not $config["showprs"] and event_type = "pull_request"
```

Comment:
All these if statements seem fairly messy, could you combine them into a loop and a hash? (Also, you need to use `==` for comparison - how did this ever work?)